Create README.md (#4)
chamaileon-sdk/serverless-example: README.md

# serverless-example

You can try the serverless chamaileon email editor plugin with the `serverless-editor-demo.html` file, provided in this repository.

> In the demo application please copy yor API key to get an accessToken, then you can initialize Chamaileon SDK and open and use the email editor.

Reviewer: ```suggestion
In the demo application please copy your API key to the corresponding field in order to get an `accessToken` which is needed to initialize Chamaileon SDK (so you can open and use the email editor).
```

## Installation

### Remote host

> Please upload the demo html to your server. Please note that your API access is limited to domains specified in your environment in order to ensure the authorized usage.

Reviewer: ```suggestion
Please upload the demo HTML to your server. Please note that your API access is limited to domains specified in your environment in order to ensure the authorized usage.
```

### Local computer

> To use the demo application from your local computer, you need to run the html file from the `localhost`, e. g. with a [static-server](https://www.npmjs.com/package/static-server). Please, use ports given in your environment, if not specified, use the port 50511.

Reviewer: ```suggestion
To use the demo application from your local computer, you need to serve the HTML file from your `localhost`, e. g. with a [static-server](https://www.npmjs.com/package/static-server). Please, use ports given in your environment, if not specified, use the port 50511.
```
